shenyu-examples/shenyu-examples-dubbo/shenyu-examples-apache-dubbo-service-annotation/src/main/java/org/apache/shenyu/examples/apache/dubbo/service/annotation/impl/DubboTestServiceImpl.java

```diff
@@ -34,7 +34,7 @@
 /**
  * The type Dubbo service.
  */
-@DubboService
+@ShenyuDubboService("demo")
 public class DubboTestServiceImpl implements DubboTestService {
```

Comment:
rollback this

Reply:
Why did not this be rollbacked? It seems that Dubbo only resolves the direct annotation, not composite annotation.